README.md

```diff
@@ -78,47 +78,29 @@ You should create a new user account to whom the virtual mailboxes will belong.
-
-**IMPORTANT NOTE:** if you have a database called "vexim" in MySQL or PostgreSQL, you should back it up first. The SQL command files assume that the database with that name does not exist and has to be created. You can edit the files to use a different database name, but you are still strongly advised to make a backup of the "vexim" database – just in case.
 
 ##### MySQL:
-This ditribution contains a file "vexim2/setup/mysql.sql". This file provides the database schema used by vexim. You will have to import it into MySQL, but before that, some changes must be made to this file in order for it to work.
+This distribution contains a file "vexim2/setup/mysql.sql". This file provides the database schema used by vexim. You will have to import it into MySQL, like this:
 
-Open this file in a text editor and look for the word "CHANGE". You will find the following two lines not far from the top (lines 15-16):
-```mysql
-uid              smallint(5)   unsigned  NOT NULL  default 'CHANGE',
-gid              smallint(5)   unsigned  NOT NULL  default 'CHANGE',
 ```
-Replace "CHANGE" with the uid and gid of your freshly created "vexim" user (see the System user step). This will be the default user to own mailbox files for new domains.
-
-Further below you will find the following section:
-```mysql
---
--- Privileges:
---
-GRANT SELECT,INSERT,DELETE,UPDATE ON `vexim`.* to "vexim"@"localhost"
-    IDENTIFIED BY 'CHANGE';
-FLUSH PRIVILEGES;
+# mysql -u root -D YOUR_DATABASE_NAME -p < vexim2/setup/mysql.sql
 ```
-Replace the word "CHANGE" with a secure password - It will be used both by the Exim MTA and the web interface to access the database.
 
-If the crypt() function on your system produces DES hashes, you will also have to uncomment the appropriate lines, as noted by the comments, at the end of mysql.sql.
-
-With the necessary changes made, you should run the following command line to initialize the database:
+Where `YOUR_DATABASE_NAME` is the name of an empty database you have created for vexim. If you want the script to create the database for you and set up access to it, this is also doable: just open it in a text editor, and find a commented out block which begins with `-- CREATE DATABASE` near the top of the file. This block is documented just above it, so you may uncomment it, apply the changes you want and save the file. With the necessary changes made, you should run the following command line to initialize the database:
 ```
 # mysql -u root -p < vexim2/setup/mysql.sql
 ```
 
+**NOTE:** if the `crypt()` function on your system does not support SHA-512 hashes, you will have to change the siteadmin password hash at the bottom of the file. Perhaps the easiest way is to set it to the following value: `$1$qZc7ANMc$h07fKA10jQQmJ33fzlJ3Z0`.
+
```

Comment:
You mean the initial password is a MD5-hash which will then be changed 
to SHA512 when the siteadmin changes his password? So we only need to 
deal with encryption schemes in the configuration-file, which makes the 
installation process easier.

On 2016-08-03 13:10, Rimas Kudelis wrote:
> In README.md [1]:
> 
>>  ```
>>  # mysql -u root -p < vexim2/setup/mysql.sql
>>  ```
>> 
>> +**NOTE:** if the `crypt()` function on your system does not support
> SHA-512 hashes, you will have to change the siteadmin password hash at
> the bottom of the file. Perhaps the easiest way is to set it to the
> following value: `$1$qZc7ANMc$h07fKA10jQQmJ33fzlJ3Z0`.
>> +
> 
> @Udera [2], I'm thinking we should probably change the password hash
> supplied in mysql.sql back to MD5 scheme. This would allow us to drop
> this paragraph as well, by making the hash almost universally
> acceptable. I don't think anyone would go around changing the hashing
> scheme in the config from the default SHA-512 to MD5 just because
> they'd inspect the SQL file and find this string there. I tend to
> think if people won't know better, they'll just leave that option as
> is.
> 
> --
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub [3], or mute the
> thread [4].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/vexim/vexim2/pull/179#discussion_r73318074
> [2] https://github.com/Udera
> [3]
> https://github.com/vexim/vexim2/pull/179/files/63b23b1653d04d1693eac13720b6d4e8d7edb4ec#r73318074
> [4]
> https://github.com/notifications/unsubscribe-auth/AGCFFCONclX9t7EcgQEpVV5XCnojtQGRks5qcHdCgaJpZM4JbhrU